engine/src/cmd/unit_generic.h

```diff
@@ -993,14 +811,15 @@ public: Unit();
                       const GFXColor&,
                       void *ownerDoNotDereference,
                       float phasedamage = 0 );
+
+    virtual float DealDamageToHull( const Vector &pnt, float Damage );
```

Comment:
This, too, should be declared `override`

Reply:
Done